examples/fc_network.rb

```diff
@@ -46,7 +46,7 @@
   action :create_if_missing
 end
 
-# This examples works only from API2600
+# This examples works only from API2800
```

Comment:
this is because of script issue. Please revert these kind of changes.

Reply:
Done